--- /Users/dby051225/Desktop/GB Essay 2026/Test/20260212-093015__RESC__RD4AD__baseline_cpu__seed42__debug/04_configs/orig__RESC_RD4AD_test.yaml	2026-02-12 11:45:05
+++ /Users/dby051225/Desktop/GB Essay 2026/Test/20260212-093015__RESC__RD4AD__baseline_cpu__seed42__debug/04_configs/RESC_RD4AD_test__final__20260224-145655.yaml	2026-02-24 11:46:52
@@ -1,31 +1,31 @@
 dataset:
   name: RESC
   format: folder
-  path: /home/jinan/Datasets/Medical-datasets/RESC
+  path: /Users/dby051225/Desktop/GB Essay 2026/Test/20260212-093015__RESC__RD4AD__baseline_cpu__seed42__debug/03_data/BMAD/datasets/RESC
   task: segmentation # classification or segmentation
 
-  train_batch_size: 32
+  train_batch_size: 2
   eval_batch_size: 1
-  inference_batch_size: 32
-  num_workers: 8
+  inference_batch_size: 2
+  num_workers: 0
   image_size: 224 # dimensions to which images are resized (mandatory)
   center_crop: null # dimensions to which images are center-cropped after resizing (optional)
   normalization: imagenet # data distribution to which the images will be normalized: [none, imagenet]
 
-  normal_dir: /home/jinan/Datasets/Medical-datasets/RESC/Train/train/good # name of the folder containing normal images.
-  # abnormal_dir:  /home/jinan/Datasets/Medical-datasets/RESC/Val/val/Ungood # name of the folder containing abnormal images.
-  # mask: /home/jinan/Datasets/Medical-datasets/RESC/Val/val_label/Ungood #optional
-  # normal_test_dir: /home/jinan/Datasets/Medical-datasets/RESC/Val/val/good/ # optional
-  abnormal_dir:  /home/jinan/Datasets/Medical-datasets/RESC/Test/test/Ungood # name of the folder containing abnormal images.
-  mask: /home/jinan/Datasets/Medical-datasets/RESC/Test/test_label/Ungood #optional
-  normal_test_dir: /home/jinan/Datasets/Medical-datasets/RESC/Test/test/good/ # optional
+  normal_dir: /Users/dby051225/Desktop/GB Essay 2026/Test/20260212-093015__RESC__RD4AD__baseline_cpu__seed42__debug/03_data/BMAD/datasets/RESC/train/good # name of the folder containing normal images.
+  # abnormal_dir:  data/RESC/Val/val/Ungood # name of the folder containing abnormal images.
+  # mask: data/RESC/Val/val_label/Ungood #optional
+  # normal_test_dir: data/RESC/Val/val/good/ # optional
+  abnormal_dir:  /Users/dby051225/Desktop/GB Essay 2026/Test/20260212-093015__RESC__RD4AD__baseline_cpu__seed42__debug/03_data/BMAD/datasets/RESC/test/ungood/img # name of the folder containing abnormal images.
+  mask: /Users/dby051225/Desktop/GB Essay 2026/Test/20260212-093015__RESC__RD4AD__baseline_cpu__seed42__debug/03_data/BMAD/datasets/RESC/test/ungood/label #optional
+  normal_test_dir: /Users/dby051225/Desktop/GB Essay 2026/Test/20260212-093015__RESC__RD4AD__baseline_cpu__seed42__debug/03_data/BMAD/datasets/RESC/test/good/img # optional
   extensions: null #  Type of the image extensions to read from the directory. Defaults to None.
   
   test_split_mode: from_dir # options: [from_dir, synthetic]
   test_split_ratio: 0 # fraction of train images held out testing (usage depends on test_split_mode)
   val_split_mode: same_as_test # options: [same_as_test, from_test, synthetic]
   val_split_ratio: 1 # ratio for validation
-  seed: 1800
+  seed: 42
 
   transform_config:
       train: null
@@ -57,6 +57,11 @@
   anomaly_map_mode: multiply
 
 metrics:
+  image:
+    - AUROC
+  pixel:
+    - AUROC
+    - AUPRO
   # image:
   #   - AUROC
   # pixel:
@@ -70,17 +75,16 @@
 
 visualization:
   show_images: False # show images on the screen
-  save_images: False # save images to the file system
+  save_images: True # save images to the file system
   log_images: False # log images to the available loggers (if any)
-  image_save_path: null # path to which images will be saved
+  image_save_path: /Users/dby051225/Desktop/GB Essay 2026/Test/20260212-093015__RESC__RD4AD__baseline_cpu__seed42__debug/07_figures/RESC_RD4AD_seed42_debug # path to which images will be saved
   mode: full # options: ["full", "simple"]
 
 project:
   seed: 42
-  path: ./results
-
+  path: /Users/dby051225/Desktop/GB Essay 2026/Test/20260212-093015__RESC__RD4AD__baseline_cpu__seed42__debug/03_data/BMAD/datasets/RESC
 logging:
-  logger: [] # options: [comet, tensorboard, wandb, csv] or combinations.
+  logger: [csv] # options: [comet, tensorboard, wandb, csv] or combinations.
   log_graph: false # Logs the model graph to respective logger.
 
 optimization:
@@ -100,7 +104,7 @@
   check_val_every_n_epoch: 2 # Don't validate before extracting features.
   fast_dev_run: false
   accumulate_grad_batches: 1
-  max_epochs: 200
+  max_epochs: 3
   min_epochs: null
   max_steps: -1
   min_steps: null
@@ -111,7 +115,7 @@
   limit_predict_batches: 1.0
   val_check_interval: 1.0 # Don't validate before extracting features.
   log_every_n_steps: 50
-  accelerator: auto # <"cpu", "gpu", "tpu", "ipu", "hpu", "auto">
+  accelerator: cpu # <"cpu", "gpu", "tpu", "ipu", "hpu", "auto">
   strategy: null
   sync_batchnorm: false
   precision: 32
